.vscode/settings.json

```diff
@@ -18,5 +18,10 @@
     "javascript",
     "typescript",
     "vue"
-  ]
+  ],
+  "tailwindCSS.experimental.configFile": ".nuxt/tailwind.config.cjs",
+  "files.associations": {
+    "*.ccs": "tailwindcss"
```

Comment:
This is meant to be `*.css` I think!

Reply:
What do you mean? This is to improve editor-support and taken from https://tailwindcss.nuxt.dev/tailwind/editor-support

Reply:
Editor support would be improved if the association is to CSS files - it's just misspelt here 🙂

Reply:
Ah of course - very good catch :D Fixed (: